float64 center_x
float64 center_y
float64 radius
